src/Sylius/Bundle/AdminBundle/Resources/private/js/app.js

```diff
@@ -60,9 +60,11 @@ $(document).ready(() => {
       $('select[name^="sylius_promotion[actions]"][name$="[type]"]').last().change();
     }, 50);
   });
-  $('#rules a[data-form-collection="add"]').on('click', () => {
+  $('#rules a[data-form-collection="add"]').on('click', (event) => {
+    const name = $(event.target).closest('form').attr('name');
+
     setTimeout(() => {
-      $('select[name^="sylius_promotion[rules]"][name$="[type]"]').last().change();
+      $(`select[name^="${name}[rules]"][name$="[type]"]`).last().change();
```

Comment:
The changes in this file affect the promotions also. Just a heads up to you guys, @lchrusciel, @Zales0123. It works of course, but I don't know if you 'like' it or not